.buildkite/pipeline.yml

```diff
@@ -12,4 +12,45 @@ steps:
     command: "make test"
     key: "tests"
     artifact_paths:
-      - "coverage.out"
\ No newline at end of file
+      - "coverage.out"
+
+  - label: ":sonarqube: Static Code Analysis"
+    env:
+      VAULT_SONAR_TOKEN_PATH: "kv/ci-shared/serverless/shared-analysis-token"
+    agents:
+      image: "docker.elastic.co/cloud-ci/sonarqube/buildkite-scanner:latest"
+    command:
+      - "buildkite-agent artifact download coverage.out ."
+      - "/scan-source-code.sh"
+    soft_fail: true
+    depends_on: "tests"
+
+  # Need go 1.20 in the serverless-docker-builder to enable this 
+  #- label: ":go: Build"
+  #  command: "make all"
+
+  # multiarch dockerx build
+  - group: ":docker: Build Container Images"
+    steps:
+      - label: ":docker: :seedling: Trigger Image Creation"
+        command: "make -C /agent generate-docker-images"
+        agents:
+          image: "docker.elastic.co/ci-agent-images/serverless-docker-builder:0.0.4"
+  
+  - wait:
+  
+  - label: ":argo: Update image tag for elasticsearch-k8s-metrics-adapter"
+    branches: main
+    trigger: k8s-gitops-update-image-tag
+    build:
+      env:
+        SERVICE: "elasticsearch-k8s-metrics-adapter"
+        IMAGE_TAG: "${DOCKER_IMAGE_TAG}"
+# Slack notifications have to be manually setup by the CI team for now
+# Please open an issue similar to https://github.com/elastic/ci/issues/960,
```

Comment:
I think it has changed and we are self-service now: https://buildkite.com/organizations/elastic/services.

Reply:
aah thank you for pointing that out!!